lib/measured/rails/active_record.rb

```diff
@@ -45,8 +45,8 @@ def measured(measured_class, *fields)
             precision = self.column_for_attribute(value_field_name).precision
             scale = self.column_for_attribute(value_field_name).scale
             rounded_to_scale_value = incoming.value.round(scale)
-            ## For BigDecimal#split syntax, refer http://ruby-doc.org/stdlib-2.1.1/libdoc/bigdecimal/rdoc/BigDecimal.html#method-i-split
-            if rounded_to_scale_value.split[1].size > precision
+            
+            if rounded_to_scale_value.to_s.length > precision
```

Comment:
Yes you could use a log base ten, but keep this since it's easier to understand and this will not be a bottleneck since these methods are written in C. Computers are fast.
```ruby
[12] pry(main)> Benchmark.measure { 1_000_000.times {|i| (i*14234).to_s.length } }
=> #<Benchmark::Tms:0x007f07dc2560a8 @cstime=0.0, @cutime=0.0, @label="", @real=0.256809833, @stime=0.0, @total=0.26000000000000156, @utime=0.26000000000000156>
```